draft-parecki-oauth-client-id-metadata-document.md

```diff
@@ -151,7 +151,8 @@ the client to the user in an authorization consent screen, for example the
 client name and logo.
 
 The authorization server SHOULD fetch the document indicated by the `client_id`
-to retrieve the client registration information.
+to retrieve the client registration information. The authorization server
+MUST NOT follow HTTP redirects when fetching the Client Metadata.
```

Comment:
@aaronpk perhaps something like this would work? It does duplicate some of the text in `Client Metadata`, but perhaps more clearly defines a successful response vs an error response, which I think is something we've been asked about in the past?

```suggestion
to retrieve the client registration information. A successful response MUST use the 200 OK HTTP status code and return a JSON object using the `application/json` or `application/<AS-defined>+json` content type. The authorization server MUST treat all other HTTP status codes and content types as an error response. The authorization server MUST NOT automatically follow redirects given via the Location header in responses.
```

The last sentence is perhaps a bit of a given, but the redundancy is probably wise here for clarity?